docs/proposals/21-ipv6-singlestack-local.md

```diff
@@ -261,7 +261,7 @@ Hence, gardenlet explicitly enables the corresponding kernel setting for shoot w
 
 The `docker.io` registry doesn't support pulling images over IPv6 (see [Beta IPv6 Support on Docker Hub Registry](https://www.docker.com/blog/beta-ipv6-support-on-docker-hub-registry/)).
 
-Container images from `docker.io` used on shoots and seeds are rewritten to `registry.ipv6.docker.com` if the corresponding `ipFamilies` field specifies IPv6 singe-stack.
+Container images from `docker.io` used on shoots and seeds are rewritten to `mirror.gcr.io` which supports dual-stack network access.
```

Comment:
Can you document this in https://github.com/gardener/gardener/blob/master/docs/usage/ipv6.md as well?
Also, please add a point to https://github.com/gardener/gardener/blob/master/docs/development/component-checklist.md that no images should be used from docker hub because of the well-known rate limits and the missing dual-stack access.